### ISSUE TITLE
[HTML Canvas] TextMetrics Additions

### ISSUE THREAD

**Issue by schenney-chromium** [mozilla/standards-positions#1144]
## Request for Mozilla Position on an Emerging Web Specification

* Specification title: HTML
* Specification or proposal URL (if available): https://github.com/whatwg/html/issues/10677
* Explainer URL (if available): https://github.com/Igalia/explainers/blob/main/canvas-formatted-text/text-metrics-additions.md
* Proposal author(s) (`@`-mention GitHub accounts): @schenney-chromium, @AndresRPerez12, @fserb
* Caniuse.com URL (optional): 
* Bugzilla URL (optional): 
* Mozillians who can provide input (optional):
* WebKit standards-position: 

### Other information

This proposal is moving through WHATWG and we are seeking the input of Mozilla folks. There is a Chromium implementation and portions are landing in WebKit already. Please see the issue discussion for known concerns.

**Comment by smaug----** [comment]
@ladyhavoc @jfkthame

**Comment by zcorpan** [comment]
HTML-in-canvas has progressed a bit since this was filed. Is this addition still needed?

**Comment by schenney-chromium** [comment]
Yes. The need for per-grapheme rendering control is still there, along with ways to facilitate editing. While HTML-in-Canvas covers the use cases for HTML-supported editing, it doesn't cover per-grapheme positioning because HTML does not give such low level control.

Chrome is moving forward with shipping things, which I personally am not happy about. We're also waiting for a counter proposal from Apple.
